packages/core/src/createActor.ts

```diff
@@ -106,7 +107,11 @@ export class Actor<TLogic extends AnyActorLogic>
   public _parent?: AnyActorRef;
   /** @internal */
   public _syncSnapshot?: boolean;
-  public ref: ActorRef<SnapshotFrom<TLogic>, EventFromLogic<TLogic>>;
+  public ref: ActorRef<
+    SnapshotFrom<TLogic>,
+    EventFromLogic<TLogic>,
+    EmittedFrom<TLogic>
+  >;
```

Comment:
It looks like this field (`ref`) is unused in any practical sense.  Backwards-compat thing?

Reply:
Yeah, will remove in a separate PR

Reply:
it's a public property, removing it now would be a breaking change